samples/zh-tw/Getting Started/main.css

```diff
@@ -1,26 +1,25 @@
 html {
-    background-color: #e6e9e9;
-    background-image: -webkit-linear-gradient(270deg,rgb(230,233,233) 0%,rgb(216,221,221) 100%);
-    background-image: linear-gradient(270deg,rgb(230,233,233) 0%,rgb(216,221,221) 100%);
+    background: #e6e9e9;
+    background-image: linear-gradient(270deg, rgb(230, 233, 233) 0%, rgb(216, 221, 221) 100%);
     -webkit-font-smoothing: antialiased;
 }
 
 body {
-    margin: 0 auto;
-    padding: 2em 2em 4em;
-    max-width: 800px;
+    background: #fff;
+    box-shadow: 0 0 2px rgba(0, 0, 0, 0.06);
+    color: #545454;
     font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
     font-size: 16px;
-    line-height: 1.5em;
-    color: #545454;
-    background-color: #ffffff;
-    box-shadow: 0 0 2px rgba(0, 0, 0, 0.06);
+    line-height: 1.5;
+    margin: 0 auto;
+    max-width: 800px;
+    padding: 2em 2em 4em;
 }
 
 h1, h2, h3, h4, h5, h6 {
     color: #222;
-    font-weight: 700;
-    line-height: 1.3em;
+    font-weight: 600;
```

Comment:
`font-weight: 700` is now `font-weight: 600`. It matches the others, but might be worth it to double check if the `zh-tw` font requires  the heavier weight.